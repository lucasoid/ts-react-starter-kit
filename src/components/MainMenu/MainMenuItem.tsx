import * as React from 'react';
import * as styles from './MainMenu.css';

interface MainMenuItemProps {
    children?: any;
}

export const MainMenuItem: React.FC<MainMenuItemProps> = props => (
    <li className={styles.mainMenuItem}>{props.children}</li>
);
