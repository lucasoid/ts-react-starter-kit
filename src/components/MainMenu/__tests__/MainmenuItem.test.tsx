import * as React from 'react';
import { shallow } from 'enzyme';
import { MainMenuItem } from '../MainMenuItem';

describe('MainMenuItem', () => {
    it('displays a main menu item', () => {
        let wrapper = shallow(
            <MainMenuItem>
                <a href="/faq">FAQ</a>
            </MainMenuItem>,
        );
        expect(wrapper).toMatchSnapshot();
    });
});
