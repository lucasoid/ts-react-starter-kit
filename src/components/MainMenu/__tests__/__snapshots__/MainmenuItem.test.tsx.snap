// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainMenuItem displays a main menu item 1`] = `
<li
  className="mainMenuItem"
>
  <a
    href="/faq"
  >
    FAQ
  </a>
</li>
`;
